boards/nxp/frdm_mcxn236/doc/index.rst

```diff
@@ -72,6 +72,8 @@ The FRDM-MCXN236 board configuration supports the following hardware features:
 +-----------+------------+-------------------------------------+
 | ADC       | on-chip    | adc                                 |
 +-----------+------------+-------------------------------------+
+| FLEXCAN   | on-chip    | CAN                                 |
++-----------+------------+-------------------------------------+
```

Comment:
We need to add `- can` to the board YAML as well so it runs CAN tests

Reply:
Apart from this, the PR looks good.